components/mdc/TextInput/TextField.svelte

```diff
@@ -29,7 +29,7 @@ const focus = (node) => autofocus && node.focus()
 
 <style>
 .required {
-  color: #da1414;
+  color: var(--mdc-required-input);
```

Comment:
ooooooooh-kay ok ok! i think i finally understand what you were getting at @merchako ! I've not used this CSS magic before so I was a little confused. this is what you meant, right? then in our app we would need to define the var as well, yes?

Reply:
yes, we define that in _theme.scss and presto! It makes its way into the component. If you want you can also set a backup like in our progress bar I mentioned above. Suggestion is Optional but would make things work out fo the box if we didn't bother to set the first var.
```suggestion
  color: var(--mdc-required-input, var(--mdc-theme-status-error));
```

Reply:
nice!